wmfdata/utils.py

```diff
@@ -52,10 +52,10 @@ def insert_code_toggle():
     display(HTML("""
     <form action="javascript:code_toggle()">
         <input
-          id="code_toggle"
-          type="submit"
-          value="Hide code"
-          style="font-size: 1.4em"
+            id="code_toggle"
+            type="submit"
+            value="Hide code"
+            style="font-size: 1.4em"
```

Comment:
Do we need these blank space changes or can we remove?

Reply:
I would suggest that they be maintained as they're from improper indentation. All of the changes in indentation were from where there were places with two spaces instead of four :)